src/layman/layer/filesystem/gdal.py

```diff
@@ -200,6 +200,19 @@ def get_statistics(filepath):
     return result
 
 
+def get_file_list_statistics(filepaths):
+    stats_list = [get_statistics(filepath) for filepath in filepaths]
+    bands_count = len(stats_list[0])
+    stats = []
+    for i in range(0, bands_count):
+        ongoing_band_min, ongoing_band_max = stats_list[0][i][0], stats_list[0][i][1]
```

Comment:
band_min = min(stats[band_idx][0] for stats in stats_list)
band_max = max(stats[band_idx][1] for stats in stats_list)
stats.append([band_min, band_max])